yaramo/signal.py

```diff
@@ -47,6 +57,7 @@ def __init__(
         function: SignalFunction | str,
         kind: SignalKind | str,
         side_distance: float = None,
+        states: List[SignalState] = None,
```

Comment:
is this the list of possible states of that signal?

Reply:
yes

Reply:
Wouldn't `possible_states` be a better name? `states` could also be a history of states during a simulation. Or the list of current shown states.